src/gei/Commands/MigrateArchiveRepoCommand.cs

```diff
@@ -0,0 +1,186 @@
+﻿using System;
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.Threading.Tasks;
+
+namespace OctoshiftCLI.GithubEnterpriseImporter.Commands
+{
+    public class MigrateArchiveRepoCommand : Command
```

Comment:
are there unit tests for this command?

Reply:
Part of https://github.com/github/gh-gei/issues/231